doc/source/contributing_docs.rst

```diff
@@ -0,0 +1,144 @@
+the upstream (main project) MDAnalysis repository.
+
+
+.. _create-virtual-environment-user-guide:
+
+Creating a development environment
+==================================
+
+:ref:`Create a new virtual environment <create-virtual-environment>` for the user guide. Install the required dependencies, and activate the ``nglview`` extension. We use ``nglview`` for visualizing molecules in Jupyter notebook tutorials.
+
+If using conda:
+
+    .. code-block:: bash
+
+        cd UserGuide/
+        conda env create python=3.6 -f environment.yml ==quiet
+        conda activate mda-user-guide
+        jupyter-nbextension enable nglview ==py ==sys-prefix
+
+
+If using pip:
+
+    .. code-block:: bash
+
+        cd UserGuide/
+        pip install -r requirements.txt
+        jupyter-nbextension enable nglview ==py ==sys-prefix
+
+.. _build-user-guide:
+
+Building the user guide
+=======================
+
+Navigate to the ``doc/`` directory and run ``make html``:
+
+    .. code-block:: bash
+
+        cd doc/
+        make html
+
+The HTML output will be in ``doc/build/``, which you can open in your browser of choice. The homepage is ``doc/build/index.html``.
+
+If rebuilding the documentation becomes tedious after a while, install the :ref:`sphinx-autobuild <autobuild-sphinx>` extension. 
+
+Saving state in Jupyter notebooks
+=================================
+
+One of the neat things about ``nglview`` is the ability to interact with molecules via the viewer. This ability can be preserved for the HTML pages generated from Jupyer notebooks by ``nbsphinx``, if you save the notebook widget state after execution.
+
+.. _add-changes-user-guide:
+
+Adding changes to the UserGuide
+===============================
+
+As with the code, :ref:`commit and push <adding-code-to-mda>` your code to GitHub. Then :ref:`create a pull request <create-a-pull-request>`. The only test run for the User Guide is: that your file compile into HTML documents without errors. As usual, a developer will review your PR and merge the code into the User Guide when it looks good.
+
+
+.. _autobuild-sphinx:
+
+Automatically building documentation
+====================================
+
+Constantly rebuilding documentation can become tedious when you have many changes to make. Use `sphinx-autobuild <https://pypi.org/project/sphinx-autobuild>`_ to rebuild documentation every time you make changes to any document, including Jupyter notebooks. Install ``sphinx-autobuild``:
+
+    .. code-block:: bash
+
+        pip install sphinx-autobuild
+
+Then, run the following command in the ``doc/`` directory:
+
+    .. code-block:: bash
+
+        python -m sphinx_autobuild source build
+
+This will start a local webserver at http://localhost:8000/, which will refresh every time you save changes to a file in the documentation. This is helpful for both the user guide (first navigate to ``UserGuide/doc``) and the main repository documentation (navigate to ``package/doc/sphinx``).
+
+
```

Comment:
How do you put up a copy of the user guide at https://lilyminium.github.io/UserGuide/ — just gh-pages? 

This trick might also be useful to share as it makes reviewing changes (especially with widgets) so much easier.

Reply:
Yes, I just merge the upstream branch into `gh-pages`, make the HTML files and copy it to the top level, and push to GitHub. This is much easier to set up than Travis deployment, but messier (and gets a bit tiresome if you work on the user guide for a long-term period) . Do you think it's worthwhile laying out how to deploy via Travis?